src/main/java/com/minecolonies/coremod/colony/CitizenDataView.java

```diff
@@ -51,6 +55,11 @@
     private BlockPos workBuilding;
 
     /**
+     * The 4 lines of the latest status.
+     */
+    private ITextComponent[] latestStatus = new ITextComponent[4];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)